### ISSUE TITLE
bug: `hide-action-buttons` patch does not work

### ISSUE THREAD

**Issue by S23u512** [inotia00/ReVanced_Extended#1033]
### Type

Cosmetic

### Tools used

Revancify

### Application

YouTube 18.24.37
Patches: 2.181.1

### Bug description

There are like/dislike, download and save buttons visible under the player even when set to hide in the settings, check the screenshot. This cosmetic bug started after the latest 2.181.1 patch.

### Steps to reproduce

Patch it, enable settings to hide buttons under the player, notice how buttons are still visible.

### Relevant log output

```shell
[patch_log.txt](https://github.com/inotia00/ReVanced_Extended/files/11958874/patch_log.txt)
```

### Screenshots or videos

![Screenshot_20230705_180550_ReVanced Extended](https://github.com/inotia00/ReVanced_Extended/assets/138691006/3c5055f7-2cfd-4131-bfe2-c4c5f35c7a6a)

### Device Environment

Android 13, OneUI 5.1

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.
- [X] I have written the title and contents in English.
- [X] This issue does not reproduce on unpatched YouTube or YT Music.

**Comment by S23u512** [comment]
I could try to go back to that version.

**Comment by anil-al** [comment]
And, something important and interesting, the bugs I faced changes when spoofing method changes. Even though one doesn't change these settings, after changing spoofing method, blank buttons appear in —I recall as— "incognito method" but the default method does not trigger it.

**Comment by S23u512** [comment]
I got this bug when I had version spoofing disabled, didnt test it with disbled.

**Comment by inotia00** [comment]
fixed in revanced-patches-v2.181.4
